besu/src/main/java/org/hyperledger/besu/controller/BesuControllerBuilder.java

```diff
@@ -256,6 +259,17 @@
     final EthContext ethContext = new EthContext(ethPeers, ethMessages, scheduler);
     final SyncState syncState = new SyncState(blockchain, ethPeers);
     final boolean fastSyncEnabled = syncConfig.getSyncMode().equals(SyncMode.FAST);
+
+    final Optional<EIP1559> eip1559;
+    final GenesisConfigOptions genesisConfigOptions =
+        genesisConfig.getConfigOptions(genesisConfigOverrides);
+    if (ExperimentalEIPs.eip1559Enabled
+        && genesisConfigOptions.getEIP1559BlockNumber().isPresent()) {
+      eip1559 = Optional.of(new EIP1559(genesisConfigOptions.getEIP1559BlockNumber().getAsLong()));
+    } else {
+      eip1559 = Optional.empty();
+    }
+    genesisConfig.getConfigOptions(genesisConfigOverrides).getEIP1559BlockNumber();
```

Comment:
return value ignored?  Are we relying on side effects or is this a stray line that should be deleted?

Reply:
It is a stray line that should be deleted.
Done.